bower.json

```diff
@@ -15,7 +15,7 @@
     "tests"
   ],
   "dependencies": {
-    "ros-rviz": "osrf/polymer-ros-rviz#master",
+    "ros-rviz": "EndPointCorp/polymer-ros-rviz#9c4061476657724fa8900a1ee973748e3781035d",
```

Comment:
For easy reference, here's the difference between the current and the proposed dependencies:

https://github.com/osrf/polymer-ros-rviz/compare/master...EndPointCorp:9c4061476657724fa8900a1ee973748e3781035d

In summary, it needs changes on:

* https://github.com/RobotWebTools/ros3djs
* https://github.com/jstnhuang/ros-websocket
* https://github.com/jstnhuang/ros-service